keanacobarde/on-paper
# CATEGORY DETAILS AND ROUTING ISSUES

## Description
- Ported template for CategoryDetails page.
- Set up skeleton logic for CategoryDetails page.
- Corrected navbar routing issues and index.js routing issues

## Related Issue
#10 
#8 

## Motivation and Context
This implementation is necessary for the operation of this application. 

## How Can This Be Tested?
- Pull PR branch to local machine
- Install needed dependencies
- Navigate between categories and view the Spending Limit as well as the remainder amount to ensure it updates.

## Screenshots (if appropriate):
N / A

## Types of changes
